modules/xfem/src/userobjects/CrackMeshCut3DUserObject.C

```diff
@@ -30,15 +30,22 @@ CrackMeshCut3DUserObject::validParams()
   MooseEnum growthDirection("MAX_HOOP_STRESS FUNCTION", "FUNCTION");
   params.addParam<MooseEnum>(
       "growth_dir_method", growthDirection, "choose from FUNCTION, MAX_HOOP_STRESS");
-  MooseEnum growthRate("FATIGUE FUNCTION", "FUNCTION");
-  params.addParam<MooseEnum>("growth_rate_method", growthRate, "choose from FUNCTION, FATIGUE");
+  MooseEnum growthRate("REPORTER FUNCTION", "FUNCTION");
+  params.addParam<MooseEnum>("growth_rate_method", growthRate, "choose from FUNCTION, REPORTER");
```

Comment:
I wouldn't change this in this PR, but would `growth_increment_method` be more appropriate?